### PR TITLE
services: adjust service details header to use less custom CSS

### PR DESCRIPTION
**Pull request by KKoukiou**
Use ct-card for making this card same to all other cards in cockpit.

Rely on Flex for spacing the card header elements.

### CODE REVIEW

**Comment by martinpitt** [comment]
unit-test failure:
```
# /source/pkg/systemd/service-details.jsx
#   29:10  error  'Text' is defined but never used          no-unused-vars
#   29:16  error  'TextVariants' is defined but never used  no-unused-vars
```

File: pkg/systemd/service-details.jsx [hunk 1]
```diff
@@ -623,24 +623,20 @@ export class ServiceDetails extends React.Component {
...
                 <CardHeader>
-                    <Flex className="service-top-panel" spaceItems={{ default: 'spaceItemsSm' }} alignItems={{ default: 'alignItemsCenter' }}>
-                        <CardTitle>
-                            <Text component={TextVariants.h2} className="service-name">{this.props.unit.Description}</Text>
-                            {this.state.isPinned &&
-                                <Tooltip content={_("Pinned unit")}>
-                                    <ThumbtackIcon className='service-thumbtack-icon' />
-                                </Tooltip>}
-                        </CardTitle>
+                    <Flex className="service-top-panel" spaceItems={{ default: 'spaceItemsMd' }} alignItems={{ default: 'alignItemsCenter' }}>
```
Comment (martinpitt) [inline pkg/systemd/service-details.jsx:626]: So the class is now only kept for the selectors in the tests, not for CSS any more, right?

**Review by garrett — APPROVED** [review]
:+1: LGTM!

**Comment by garrett** [comment]
Oh, is this a problem in this PR or on main?

image

**Comment by KKoukiou** [comment]
> Oh, is this a problem in this PR or on main? (This unit doesn't wrap, causing the page)
> 
This is same on main - we don't break words.

**Comment by garrett** [comment]
Yeah, confirmed: I see that on main too. (Just switched and rebuilt.)

**Pull request merged**